moped-editor/src/queries/project.js

```diff
@@ -103,7 +103,7 @@ export const SUMMARY_QUERY = gql`
       phase_name
       phase_order
     }
-    moped_entity(order_by: { entity_id: asc }) {
+    moped_entity(order_by: { entity_name: asc }) {
```

Comment:
This has the effect of pushing some obscure non-COA entities to the top of the dropdown menus. I'd rather live with this than have Smart Mobility buried at the bottom. The right we to solve this is to add an `order` column on the table that we can use for sorting.